### PR TITLE
Release 1.38.0

### PR DESCRIPTION
**Pull request by thrix**
- Added Luigi and Otto to the contributors
- Reordered release notes according to this key: ** New and possibly breaking features
    - New features
    - Bug fixes
    - Documentation updates and others

### Follow the steps below to create a new major or minor release:

- [x] Update overview.rst with new contributors since the last release
- [x] Review the release notes in releases.rst, update as needed
- [x]  Add a Release x.y.z commit, empty if needed: git commit --allow-empty -m "Release x.y.z"
- [x]  Create a pull request with the commit, ensure tests pass, merge it

### Create a new github release based on the tag above

- [ ] Move the fedora branch to point to the new release
- [ ] Tag the commit with x.y.z, push tags git push --tags
- [ ]  Create a new github release based on the tag above
- [ ]  Mention the most important changes in the name, do not include version
- [ ] Use ; as a delimiter, when multiple items are mentioned in the name
- [ ] Push the “Generate release notes” button to create the content
- [ ] Prepend the “See the release notes for the list of interesting changes.” line
- [ ] Publish the release, check Fedora pull requests, make sure tests pass and merge

### Finally, if everything went well

- [ ] Close the corresponding release milestone
- [ ] Once the non development copr build is completed, move the quay branch to point to the release commit as well to build fresh container images.

### Handle manually what did not went well

- [ ] If the automation triggered by publishing the new github release was not successful, publish the fresh code to the pypi repository manually using make wheel && make upload
- [ ] If there was a problem with creating Fedora pull requests, you can trigger them manually using /packit propose-downstream in any open issue.

### CODE REVIEW

File: docs/overview.rst [hunk 1]
```diff
@@ -777,7 +777,7 @@ James Molet, Cristian Le, Lili Nie, Martin Čermák, Michael Vogt,
 Qinghua Cheng, Michael Engel, Anatoli Babenia, Colin Walters,
 Link Dupont, Mario Casquero, Martin Klusoň, Pavel Holica,
 Otto Šabart, Ismail Ibrahim Quwarah, Sergei Petrosian, Tom
-Koscielniak and Han Han.
+Koscielniak, Han Han, Luigi Pellecchia and Otto Sabart.
```
Comment (martinhoyer) [inline docs/overview.rst:780]: Otto is already there

File: docs/releases.rst [hunk 2]
```diff
@@ -8,17 +8,12 @@
...
-not called during the test, the shell test framework behavior
-remains the same - the test script exit code still has an impact
-on the main test result. See also
-:ref:`/stories/features/report-result`.
+In order to prevent dangerous commands to be unintentionally run
+on user's system, the :ref:`/plugins/provision/local` provision
+plugin now requires to be executed with the ``--feeling-safe``
+option or with the environment variable ``TMT_FEELING_SAFE`` set
+to ``True``. See the :ref:`/stories/features/feeling-safe` section
+for more details and motivation behind this change.
```
Comment (psss) [inline docs/releases.rst:16]: I think the check result note should go here once it's merged. It's one of the most important changes.

File: docs/overview.rst (hunk 1 above)
Comment (thrix) [reply docs/overview.rst:780]: what did my grep do?  :D

Comment (thrix) [reply docs/overview.rst:780]:
```suggestion
Koscielniak, Han Han and Luigi Pellecchia.

**Comment by thrix** [comment]
Release proposal name:

```
Test checks affect results; Local execution feeling safe; Image Mode support; Subresults support in tmt-report-result
```

**Review by psss — APPROVED** [review]
Looks good!

**Review by martinhoyer — APPROVED** [review]

**Pull request merged**